cognite_toolkit/cdf_tk/load/_resource_loaders.py

```diff
@@ -340,6 +347,150 @@ def delete(self, ids: SequenceNotStr[str]) -> int:
+    ) -> int:
+        zip_path = Path(root_dir.parent / f"{external_id}.zip")
+        root_length = len(root_dir.parts)
+        with ZipFile(zip_path, "w") as zipfile:
+            for file in root_dir.rglob("*"):
+                if file.is_file():
+                    zipfile.write(file, "/".join(file.parts[root_length - 1 : -1]) + f"/{file.name}")
+        file_info = self.client.files.upload_bytes(
+            zip_path.read_bytes(),
+            name=f"{external_id}.zip",
+            external_id=external_id,
+            overwrite=True,
+            data_set_id=data_set_id,
+        )
+        zip_path.unlink()
+        return cast(int, file_info.id)
+
+    def create(self, items: Sequence[FunctionWrite]) -> FunctionList:
+        items = list(items)
+        created = FunctionList([], cognite_client=self.client)
+        status = self.client.functions.status()
+        if status.status != "activated":
+            if status.status == "requested":
+                print("  [bold yellow]WARNING:[/] Function service activation is in progress, skipping functions.")
+                return FunctionList([])
+            else:
+                print(
+                    "  [bold yellow]WARNING:[/] Function service is not activated, activating and skipping functions..."
+                )
+                self.client.functions.activate()
+                return FunctionList([])
+        for item in items:
+            function_rootdir = Path(self.build_path / (item.external_id or ""))
+            if item.metadata is None:
+                item.metadata = {}
+            item.metadata["cdf-toolkit-function-hash"] = calculate_directory_hash(function_rootdir)
+            try:
+                file_id = self._zip_and_upload_folder(
+                    root_dir=function_rootdir,
+                    external_id=item.external_id or item.name,
+                    data_set_id=None,  # TODO: support data_set for function upload
+                )
+                created.append(
+                    self.client.functions.create(
+                        name=item.name,
+                        external_id=item.external_id or item.name,
+                        # data_set_id=item.data_set_id,
+                        file_id=file_id,
+                        function_path=item.function_path or "./handler.py",
+                        description=item.description,
+                        owner=item.owner,
+                        secrets=item.secrets,
+                        env_vars=item.env_vars,
+                        cpu=cast(Number, item.cpu),
+                        memory=cast(Number, item.memory),
+                        runtime=item.runtime,
+                        metadata=item.metadata,
+                        index_url=item.index_url,
+                        extra_index_urls=item.extra_index_urls,
+                    )
+                )
+            except Exception:
+                pass
```

Comment:
This will silently pass error. Suggest you make it propagate, it will be caught in `_create_resources` in the parent class which handles it.